polyfill/test/absolute.mjs

```diff
@@ -273,6 +273,11 @@ describe('Absolute', () => {
       equal(`${Absolute.from('+0019761118T152330.1+00:00')}`, '1976-11-18T15:23:30.100Z');
       equal(`${Absolute.from('+0019761118T152330.1+0000')}`, '1976-11-18T15:23:30.100Z');
     });
+    it('24:00 to mean next day midnight', () => {
```

Comment:
Let's also test that "24:01" doesn't work.

Reply:
Good call, that actually was accepted in cases where the time was ignored (Date, YearMonth, MonthDay). Fixed now.